src/app/common/SummaryView.vue

```diff
@@ -42,15 +42,15 @@ const emit = defineEmits<{
 
 <style lang="scss" scoped>
 :where(.summary-slideout) :deep(.app-view-title-bar) {
-  h1, h2, h3, h4, h5, h6 {
+  h1, h2 {
     --icon-before: url('@/assets/images/icon-wifi-tethering.svg');
   }
 }
 .summary-slideout :deep(.app-view-title-bar) {
   display: flex;
   // Accounts for the absolutely-positioned close button
   margin-right: calc($kui-space-30 + 24px);
-  h1, h2, h3, h4, h5, h6 {
+  h1, h2 {
```

Comment:
We only use icons currently on h1 and h2s, but this is also a temporary fix for a little CSS bug where we are also putting icons on the h3's inside the drawer. We'll before moving to use KSlideout's title slot soon, which will make this easier. Happy to leave this out and wait for the KSlideout upgrade also, I'm easy.